engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -387,11 +411,13 @@ UnexpectedTypeException getNoValidatorFoundForTypeException(String constraintTyp
 	IllegalArgumentException getIllegalArgumentException(String message);
 
 	@Message(id = 118, value = "Unable to cast %s (with element kind %s) to %s")
-	ClassCastException getUnableToNarrowNodeTypeException(String actualDescriptorType, ElementKind kind, String expectedDescriptorType);
+	// TODO
+	ClassCastException getUnableToNarrowNodeTypeException(@FormatWith(ClassObjectFormatter.class) Class<?> actualDescriptorType, ElementKind kind,
+			@FormatWith(ClassObjectFormatter.class) Class<?> expectedDescriptorType);
 
 	@LogMessage(level = INFO)
 	@Message(id = 119, value = "Using %s as parameter name provider.")
-	void usingParameterNameProvider(String parameterNameProviderClassName);
+	void usingParameterNameProvider(@FormatWith(ClassObjectFormatter.class) Class<ParameterNameProvider> parameterNameProviderClass);
```

Comment:
Sure. There are several like that. But the caller is casting the argument to Class<ParameterNameProvider>, that's why I did it like that. Maybe we should fix the caller?